packages/flutter/lib/src/painting/flutter_logo.dart

```diff
@@ -221,9 +210,9 @@ class FlutterLogoDecoration extends Decoration {
   @override
   String toString([String prefix = '', String prefixIndent ]) {
     final String extra = _inTransition ? ', transition $_position:$_opacity' : '';
-    if (swatch == null)
+    if (lightColor == null || darkColor == null)
       return '$prefix$runtimeType(null, $style$extra)';
```

Comment:
you can remove this if and its branch, it doesn't matter any more (it was just trying to prevent a null deref before)